### PR TITLE
fixed deprication warnings // added some missing semicolons

### PR DESCRIPTION
**Pull request by AdriVanHoudt**

### CODE REVIEW

**Comment by AdriVanHoudt** [comment]
@alltom any way this can be merged or comment on it?

**Comment by alltom** [comment]
I'm really sorry! I switched to Inbox and haven't gotten all my e-mail filters back in order yet! (Thanks for the bump. Otherwise, it might've taken me even longer to see this. :fearful:)

I will review and merge this ASAP. :) I really appreciate the patch.

**Comment by AdriVanHoudt** [comment]
NP inbox is awesome! An thanks for the review!

File: src/Panel.js [hunk 1]
```diff
@@ -28,12 +28,12 @@
...
-	var Main          = require("../main");
-	var PanelManager  = brackets.getModule("view/PanelManager");
-	var Resizer       = brackets.getModule("utils/Resizer");
-    var Strings       = require("./strings")
+	var EditorManager 		= brackets.getModule("editor/EditorManager");
+	var Inspector     		= brackets.getModule("LiveDevelopment/Inspector/Inspector");
+	var Main          		= require("../main");
+	var WorkspaceManager  = brackets.getModule("view/WorkspaceManager");
+	var Resizer       		= brackets.getModule("utils/Resizer");
+	var Strings       		= require("./strings");
```
Comment (alltom) [inline src/Panel.js]: Align with spaces, not tabs, with only one space after "WorkspaceManager".

Comment (AdriVanHoudt) [inline src/Panel.js]: ok i'll see when I can get to it

**Comment by alltom** [comment]
Cool, looks good to me. I'll merge as soon as that one comment is addressed. Thanks a ton! That warning has irked me for a while. :)

**Comment by AdriVanHoudt** [comment]
looks better :tada:

**Comment by alltom** [comment]
Thanks a ton. :)

**Comment by AdriVanHoudt** [comment]
Glad to help! Any eta on a next release?

**Pull request merged**